Helper/Data.php

```diff
@@ -214,8 +214,11 @@ public function getPriceThreshold()
             'payment/divido_financing/price_threshold',
             \Magento\Store\Model\ScopeInterface::SCOPE_STORE
         );
+        $options = [
+            'flags' => FILTER_FLAG_ALLOW_FRACTION
+        ];
 
-        return $threshold;
+        return filter_var($threshold, FILTER_SANITIZE_NUMBER_FLOAT, $options);
```

Comment:
I think this is a good idea! However during some simple testing I did I found If the users would input a "float" number with a decimal "," caracter, like;
"12,34" this would result in the number produced by this to be "1234", replacing all decimals with periods would solve this.

Something like;

```suggestion
        return filter_var(
            str_replace(',', '.', strval($threshold)),
            FILTER_SANITIZE_NUMBER_FLOAT,
            $options
        );
```

Reply:
You are absolutely right. My initial thought was that we don't use commas here for decimals, but didn't see the bigger picture. I stand corrected and will amend right away.

Reply:
I'm a much bigger fan of keeping the data at rest in a consistent state instead of manipulating it in the presentation layer, but since David told me there is no way to impose any rules on the input, then I guess this is the lesser evil :)